### ISSUE TITLE
LinkedIn unable to connect

### ISSUE THREAD

**Issue by georgiaevans** [engram-design/SocialPoster#52]
### Describe the bug

After following the docs for an organisation, LinkedIn is unable to connect - no error shown. 

Community Management API request has been granted. 

The URL generated to authorise contains scopes which are not listed in the Auth tab in the LinkedIn app, specifically r_liteprofile and r_emailaddress. 

r_basicprofile is listedin the Auth tab of the LinkedIn app. I have tried changing these in the URL, which took us a bit further where I was asked to allow access, but this did not connect to LinkedIn through the CMS.

### Steps to reproduce

1. Followed the docs for connecting an organisation to LinkedIn
2. Clicked connect in the CMS

### Craft CMS version

3.9.10

### Plugin version

2.3.4

### Multi-site?

No

**Comment by engram-design** [comment]
Can you share what Products you have enabled for your LinkedIn app?

**Comment by georgiaevans** [comment]
I just have the Community Management API and I'm unable to request access to any others as the Community Management API requires it to be the only product on the application

**Comment by engram-design** [comment]
Yup, you can't do anything else with "Community Management API" and if you didn't pick that at the start, you'd have to start again.

But, also just realised this is for Craft 3, which is very likely outdated now. I'll look into it further.
